src/components/AddBookComponent.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './Component.css';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { addBooks } from './redux/books/booksSlice';
+
+const AddBookComponent = ({ input, inputFunc }) => {
+  const dispatch = useDispatch();
+  const currentDate = new Date();
+  const itemId = currentDate.getTime();
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(addBooks({
+      title: input,
+      author: 'Mayito Ismail',
+      item_id: itemId,
+      category: 'fiction',
```

Comment:
-[ ] Based on project requirements, kindly make sure your component  has local state for `title` and `author`, which will make it possible to add the two from user input. From your implementation, `author` for example is already hardcoded and repeats every time a new entry is made which should not be the case.

### Sample expected code:
```
import React from "react"
import { useDispatch } from "react-redux"
import { addBook } from "path/to/booksSlice"

function NewBookForm() {
  const [title, setTitle] = useState("")
  const [author, setAuthor] = useState("")
  const books = useSelector((state) => state.books.books)
  const dispatch = useDispatch()

  // Other logic that controls form state

  return (
    <div>
      {/* Rest of form JSX */}
      <Button onClick={() => dispatch(addBook({ title, author }))}>
        Add book
      </Button>
    </div>
```

### Error output:
![image](https://github.com/ismayito/Bookstore-react-app/assets/50990925/09c542c2-d119-4526-8150-8d5258e7baf6)